README.md

```diff
@@ -102,6 +102,22 @@ More code examples are available [here](https://github.com/clauderic/react-sorta
 | useWindowAsScrollContainer | Boolean  | `false` | If you want, you can set the `window` as the scrolling container                                                                                                                                                             |
 | hideSortableGhost          | Boolean  | `true`  | Whether to auto-hide the ghost element. By default, as a convenience, React Sortable List will automatically hide the element that is currently being sorted. Set this to false if you would like to apply your own styling. |
 | lockToContainerEdges       | Boolean  | `false` | You can lock movement of the sortable element to it's parent `SortableContainer`                                                                                                                                             |
+| lockOffset                 | `OffsetValue`\* \| [`OffsetValue`\*, `OffsetValue`\*] | `"50%"` | The offsets to use when locking the movement of the sortable element. **See below for more info**. *This property has not effect of `lockToContainerEdges` is `false`.*         |
+
+\* `OffsetValue` is either a finite `Number` or a `String` made-up of a number and a unit (`px` or `%`).
+Examples: `10` (is the same as `"10px"`), `"50%"`
+
+##### lockOffset
+A single value is equivalent to a pair that same value (`"50%"` <=> `["50%", "50%"]`).
+
+```
+ +-------------------------------+        !   Sortable List                 !
+ |      lockOffset[0] = "50%" ↕  |        !+-------------------------------+!
+=|= Item 1 ========================       !|                               |!
+!|                               |!       =|= Item 5 ========================
+!+-------------------------------+!        |      lockOffset[1] = "50%" ↕  |
+!   Sortable List                 !        +-------------------------------+
+```
```

Comment:
I wanted to put an image but I didn't find a way to do it without adding the image to this repo.
But yeah I'll remove this section :smile: